policyengine_us/variables/gov/states/il/tax/income/exemptions/il_personal_exemption.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class il_personal_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IL personal exemption amount"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
+
+    def formula(tax_unit, period, parameters):
+        il_is_personal_exemption_eligible = tax_unit(
```

Comment:
test

Reply:
I thought that we had agreed that `il_personal_exemption` doesn't require a test because it's directly dependent on `il_personal_exemption_eligibility_status`, which is currently being tested. Should I create a test anyways?

Reply:
Sorry just seeing this - my mistake earlier saying we don't need one, codecov is complaining so please add a simple one.